docs/source/index.rst

```diff
@@ -5,3 +5,4 @@
    :caption: Project Documentation
```

Comment:
Doesn't let me comment, but there is an inclusion portion at line 1 that is not necessary since there is no home page for this documentation.

Reply:
This should point to something like the README or direct Gen-Gherkin page. How you added just puts it to the table of contents in the side bar.

Reply:
@bunchw still need to modify the landing page of documentation to either include the Gen-Gherkin page or the README. You just have the table of contents in the side bar displaying right now.